src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String statistic = readFromFile(fromFileName);
+        reportData(statistic, toFileName);
```

Comment:
reportData should be divided into 2 methods - getReport(), writeToFile() 
```suggestion
        String data = readFromFile(fromFileName);
        String report = getReport();
        writeToFile(report, toFileName);
```